src/oscontainer.py

```diff
@@ -117,6 +119,15 @@ def oscontainer_build(containers_storage, src, ref, image_name_and_tag,
         print("Copying ostree commit into container: {} ...".format(rev))
         run_verbose(["ostree", "--repo=" + dest_repo, "pull-local", src, rev])
 
+        # Copy commitmeta.json in to the oscontainer at /
+        commitmeta_source = os.path.join(
+            os.getcwd(), 'builds', ostree_version, 'commitmeta.json')
```

Comment:
In the general case the build isn't just the ostree version though, since we support changing the image without the ostree; this will *usually* work but will fail if the previous build has been pruned, etc.

I think the cleanest fix is to have `oscontainer build` accept a build ID (which gets slightly ugly to do while still supporting backwards compatibility), maybe make it optional like `cosa oscontainer build --buildid=blah` and that makes the positional args not required or something except for `name`?

Reply:
This is exactly what I was looking for. Thank you @cgwalters!

Reply:
Updated.

Reply:
Though made it required. Fixing this now.

Reply:
Fixed